content/sensu-core/2.0/reference/entities.md

```diff
@@ -4,82 +4,355 @@ description: "The entities reference guide."
 
-### Creating a proxy entity
+deregister   | 
+-------------|------ 
+description  | If the entity should be removed when it stops sending keepalive messages. 
+required     | false 
+type         | boolean 
+default      | false
+example      | {{< highlight shell >}}"deregister": false {{</ highlight >}}
 
-To create a proxy entity, a check's ProxyEntityID (formerly `source`) can be provided in the check definition or published as a check result to the TCP/UDP socket. If a check result includes a ProxyEntityID attribute, a proxy entity will be created, and the check result will be stored under the newly created entity.
+deregistration  | 
+-------------|------ 
+description  | A map containing a handler name, for use when an entity is deregistered. 
+required     | false 
+type         | [Deregistration][2] 
+example      | {{< highlight json >}}
+  "deregistration": {
+    "handler": "email-handler"
+}
+{{</ highlight >}}
 
-### Viewing
+keepalive_timeout  | 
+-------------|------ 
+description  | The time in seconds until an entity keepalive is considered stale. 
+required     | false 
+type         | integer 
+default      | 120
+example      | {{< highlight shell >}}"keepalive_timeout": 120 {{</ highlight >}}
 
-To view all the entities that are currently configured for the cluster, enter:
+organization | 
+-------------|------ 
+description  | The Sensu RBAC organization that this entity belongs to.
+required     | false 
+type         | string 
+example      | {{< highlight shell >}}
+  "organization": "default"
+{{</ highlight >}}
 
-{{< highlight shell >}}
-sensuctl entity list
-{{< /highlight >}}
+environment  | 
+-------------|------ 
+description  | The Sensu RBAC environment that this entity belongs to.
+required     | false 
+type         | string 
+default      | current environment value configured for `sensuctl` (ie `default`) 
+example      | {{< highlight shell >}}
+  "environment": "default"
+{{</ highlight >}}
 
-If you want more details on an entity, the `info` subcommand can help you out.
-
-> sensuctl entity info entity.localhost
-{{< highlight shell >}}
-=== entity.localhost
-ID:                     entity.localhost
-Class:                  agent
-Subscriptions:          system
-Last Seen:              2018-02-21 14:32:16 -0600 CST
-Hostname:               entity.localhost
-OS:                     darwin
-Platform:               darwin
-Platform Family:        
-Platform Version:       10.13.3
-Auto-Deregistration:    false
-Deregistration Handler:
-{{< /highlight >}}
+extended_attributes | 
+-------------|------ 
+description  | Custom attributes to include as with the entity, that appear as outer-level attributes.
+required     | false 
+type         | byte 
+example      | {{< highlight shell >}}"team":"ops"{{</ highlight >}}
 
-### Management
+redact       | 
+-------------|------ 
+description  | List of items to redact from log messages. 
```

Comment:
We should probably specify that when using this attribute, the default values get overridden and not appended.

Reply:
Oh, that is something I wasn't aware of - will add a note.